docs/overview.md

```diff
@@ -0,0 +1,872 @@
+# Introduction
+
+## Overview
+
+The MongoDB Atlas SQL ODBC driver supports connections from a SQL compliant
+client, enabling you to query your data in MongoDB with SQL queries.
+
+### Features
+
+Key Features: Native object and array support.
+
+Compatibility: The MongoDB Atlas SQL ODBC driver is compatible with MongoDB versions greater
+than or equal to 6.0.6.
+
+## System Requirements
+
+### Hardware Requirements
+
+The MongoDB Atlas SQL ODBC driver has no hard system requirements itself. Any system
+capable of running a modern SQL tool (such as Power BI, DBeaver) is capable of running
+the MongoDB Atlas SQL ODBC driver.
+
+### Software Requirements
+
+#### Supported Operating Systems
+
+The MongoDB ODBC driver is compatible with Windows x86_64 architecture, linux x86_64,
+and linux arm64 systems.
+
+#### Dependencies
+
+When querying a MongoDB standalone server or cluster (not [Atlas SQL](https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/) powered by [Atlas Data Federation](https://www.mongodb.com/docs/atlas/data-federation/)), the accompanying
+`mongosqltranslate.dll` or `libmongosqltranslate.a` library is required to be colocated with the driver (`mongoodbc.dll` or `libatsql.so`).
+
+##### `libmongosqltranslate/mongosqltranslate` Location
+
+For Linux, the location of `libmongosqltranslate` will be wherever you extracted the download artifacts, in the `bin` dirctory.
+For Windows, the default location is `C:\Program Files\MongoDB\Atlas SQL ODBC Driver\bin`.
+
+On linux system, `unixodbc` is required.
+
+## Installation
+
+### Installation Steps
+
+#### Windows Installation
+
+##### Prerequisites
+
+[Microsoft Visual C++ Redistributable 17.0](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170) or higher
+is required for the proper C and C++ (MSVC) runtime libraries to be present. You can find a link to an installer for your system [here](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#latest-microsoft-visual-c-redistributable-version).
+
+1. [Download the `MSI`](TODO).
+2. Install and Configure the ODBC Driver.
+To install the ODBC driver, run the installation file that you downloaded
+and open the Setup Wizard. Follow the steps in the Setup Wizard.
```

Comment:
Step one says `(TODO)`. Is this on purpose?

Reply:
Ah no, I need to delete this part. Good catch!